src/osd/PrimaryLogPG.cc

```diff
@@ -4248,8 +4351,32 @@ int PrimaryLogPG::trim_object(
     ctx->delta_stats.num_object_clones--;
     if (coi.is_cache_pinned())
       ctx->delta_stats.num_objects_pinned--;
-    if (coi.has_manifest())
+    if (coi.has_manifest()) {
+      set<uint64_t> refs;
+
+      for (auto p : snapset.clones) {
```

Comment:
Something's wrong with the spacing here.

Reply:
OK